appt-org/appt-website
# Fix linter to check PR's

#### What does this PR do?
Linter was broken (possibly because of the many md files), so added it to check the components again

#### How do you test this PR?
You can run `npm run lint` or `npm run lint:fix` and it should add a linter check to the PR



#### Checklist

- [ ] Works on Safari, Chrome, Firefox & Edge
- [ ] Works on different screen sizes (responsive)
- [ ] Scores 100 in [Lighthouse/PageSpeed Insights](https://pagespeed.web.dev) on SEO, accessibility and best practices, and >= 90 on performance
- [ ] Does not contain tracking cookies
- [ ] Are migrations added?
- [ ] Is the README.md updated?

**Accessibility checklist**

- Scalable text
- Dark and light mode
- Contrast
- Focus management
- Aria attributes
